sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/mapper/ChangeStreamRecordMapper.java

```diff
@@ -253,6 +255,8 @@ private DataChangeRecord toDataChangeRecord(
         valueCaptureTypeFrom(row.getString(VALUE_CAPTURE_TYPE_COLUMN)),
         row.getLong(NUMBER_OF_RECORDS_IN_TRANSACTION_COLUMN),
         row.getLong(NUMBER_OF_PARTITIONS_IN_TRANSACTION_COLUMN),
+        row.getString(TRANSACTION_TAG),
```

Comment:
Are these always present or can they be `null`? If the latter is true, you have to do `row.isNull(TRANSACTION_TAG) ? null : row.getString(TRANSACTION_TAG)`

Reply:
Yes these are always present now

Reply:
Sorry, just to make sure I understand, there will always be a transaction tag?

Reply:
Yep, there will always be transaction tag field (might be empty string) in the output